### ISSUE TITLE
Jump to search terms or rules

### ISSUE THREAD

**Issue by Tux** [choroba/pm-cb#13]
Using cbstream inside hexchat, what I miss is to be able to jump (in history) to the first message line where text appeared that I highlighted by rules (like on my alias or on several specific subjects, like CSV or HP-UX
When away from CB for longer (hours, days, weeks), it would be awesome to jump to those lines skipping the rest of the chatter

**Comment by choroba** [comment]
BTW, have you tried right-clicking in the message history?

**Comment by Tux** [comment]
Yes, but that does neither include highlighting, nor jump to next highlighted line. I can search, but I do not want to enter a complicated search string like merijn|tux|csv|hp[-/]?ux and tick the ignore case field to find things I should have seen
